tests/Functional/PluginManagerCommandsTest.php

```diff
@@ -17,26 +17,42 @@ class PluginManagerCommandsTest extends TestCase
 
     protected const HELLO_COMMAND = 'hello';
     protected const PLUGIN_NAME = 'terminus-plugin-example';
+    protected const PLUGIN_NAMES = [
```

Comment:
Consider more specific names to differentiate PLUGIN_NAME and PLUGIN_NAMES by what they are for / how they are used. Optional.

Reply:
Done.